src/Commands/Command.php

```diff
@@ -252,6 +259,16 @@ public function getName()
         return $this->name;
     }
 
+    /*
```

Comment:
This needs to be `/**` for docblock to pick it up correctly.

Also, check the number of spaces and compare it with the others.
e.g.:
```php
/**
 * There is an extra space before the *s on these lines!
 */
```